Configure

```diff
@@ -383,6 +383,7 @@ my @disablables = (
     "shared",
     "siphash",
     "sm2",
+    "sm2err",
```

Comment:
Hummmm...  where did you find anything that this would affect?  I just grepped in master, and I see...  well, nothing:

```
: ; git grep -i sm2err
crypto/sm2/sm2_err.c:#include <openssl/sm2err.h>
include/openssl/err.h:# define SM2err(f,r) ERR_PUT_error(ERR_LIB_SM2,(f),(r),OPENSSL_FILE,OPENSSL_LINE)
include/openssl/sm2err.h:#ifndef HEADER_SM2ERR_H
include/openssl/sm2err.h:# define HEADER_SM2ERR_H
```

Reply:
It impacts tests; there's a generated file (buildtest_sm2err.c)

Reply:
I think it should be folded into NO_SM2, but I wanted to get this out before the freeze.

Reply:
Actually, those that include `*err` shouldn't be generated at all.  Thanks for the pointer...  and please remove `"sm2err"`

Reply:
Will do. I'm also seeing buildtest_bioerr.c, buildtest_sslerr.c, etc.

Reply:
See #5678